homeassistant/components/metoffice/__init__.py

```diff
@@ -58,7 +58,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
 
     # Fetch initial data so we have data when entities subscribe
     await metoffice_coordinator.async_refresh()
-    if metoffice_data.now is None:
+    if metoffice_data.now_3hourly is None or metoffice_data.now_daily is None:
```

Comment:
Is this check even needed?  I realize it was there before this PR, but is it required for setting up the entities?  If a failed update only leads to unavailable data, then it can be removed.

Reply:
Probably not.

Reply:
I'd leave it as is for this PR.